Pass along gaze watch parameters so that there is more control over file watching.

robblovell opened the pull request:
The use case is described in issue #5

alexgorbatchev commented on index.js:
```diff
@@ -16,7 +17,8 @@ function startWatching({ watch, match, exec }, { verbose }) {
     match = match || [ '**/*' ];
     const prefix = [ list('watch', watch), list('match', match), list('exec', exec) ].join(' ');
-    const gaze = new Gaze(watch);
+    console.log(JSON.stringify(options.gazeOptions));
```
Could you please use `log()` function instead of `console.log()`? I would like to keep ability to have it completely silent.

robblovell replied:
will do.

robblovell replied:
I will remove this completely.

alexgorbatchev commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "run-when-changed",
-  "version": "2.0.2",
+  "version": "2.1.0",
```
I can rev the version and create a git tag at the same time when I publish.

robblovell replied:
okay, I will move this back..

robblovell commented:
Okay, I have updated the code based on your suggestions. 
1. Removed console.log
2. moved back to version 2.0.2 for the build system to do its magic.

robblovell commented:
Should this pull request be merged? The asked for changes have been made.

Cheers!
